Makefile.cache

```diff
@@ -310,13 +310,17 @@ endef
 # Read from the cache
 RCACHE_OPTIONS := cache rcache rwcache
 define CHECK_RCACHE_ENABLED
-$(if $(and $(filter $(RCACHE_OPTIONS),$(SONIC_DPKG_CACHE_METHOD)),$(filter-out none,$($(1)_CACHE_MODE))),enabled)
+$(if $(and $(filter $(RCACHE_OPTIONS), \
+					$(or $(filter-out none,$($(1)_CACHE_OVERRIDE)),$(SONIC_DPKG_CACHE_METHOD))), \
```

Comment:
It is override not or. override has higher priority than SONIC_DPKG_CACHE_METHOD.
```
$(SONIC_DPKG_CACHE_METHOD)  ->  $(if $($(1)_CACHE_OVERRIDE),$($(1)_CACHE_OVERRIDE),$(SONIC_DPKG_CACHE_METHOD))
```